lib/site_template/_config.yml

```diff
@@ -29,6 +29,15 @@ markdown: kramdown
 theme: minima
 gems:
   - jekyll-feed
-exclude:
-  - Gemfile
-  - Gemfile.lock
+
+# Exclude from processing.
+# The following items will not be processed, by default. Create a custom list
+# to override the default setting.
+# exclude:
```

Comment:
I wonder if there's anyway this list could be automatically updated with the default exclude list?